intl/msg_hash_us.h

```diff
@@ -209,11 +209,11 @@ MSG_HASH(
       )
 MSG_HASH(
       MENU_ENUM_LABEL_VALUE_AUTO_OVERRIDES_ENABLE,
-      "Load Override Files Automatically"
+      "Load Core Overrides Automatically"
```

Comment:
Bleh. I found more vomit and mess. The latest commit cleans up `git grep 'CONFIG.*N_SET'` as well as duplicate case values and previous definitions... 

@fr500 (And whitespace too, I know. Don't nag me.) I got you a present thought... Brought back "Configurations Per Core" label in the same place since I'm not sure what the stance with `show_advanced_settings` right now is. I don't see that option in the menu.

_NOTE:_
menu/menu_displaylist.c:2911
└── Your "Configurations Per Core" Label
menu/cbs/menu_cbs_ok.c:444
└── Please check this section. I commented it out because of duplicate `ACTION_OK_DL_CONFIGURATIONS_SETTINGS`. Which one do you use?

Thanks for looking.